src/js/components/Form/stories/ControlledInput.js

```diff
@@ -104,6 +107,12 @@ export const ControlledInput = () => {
                 onChange={event => setAge(event.target.value)}
               />
             </FormField>
+            <FormField required label="FileInput" name="file" pad>
```

Comment:
Add `htmlFor="file"` to FormField and `id="file"` to FileInput for accessibility